github/run.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
I believe this should be moved to `foxsec-automation`.

Reply:
Agreed -- this may cease to exist -- it's "test harness"

Reply:
removed in next commit